scripts/uboot_helper

```diff
@@ -62,6 +62,43 @@ devices = {
     },
     'TinkerBoard' : { 'rk3288' : { 'dtb' : 'rk3288-miniarm.dtb', 'config' : 'tinker-rk3288_config' }, },
   },
+
+  'NXP' : {
```

Comment:
This commit adds an empty line and the `NXP` project is placed at end instead of in alphabetical order.
All other boards in this file is listed on one line instead of dtb/config on their own lines, maybe a pre-PR or follow-up PR to reformat the file?

The board names on Rockchip/Amlogic use `-` in the board name to follow `dtb`-files, Allwinner and NXP use `_`. This also introduces uppercase in a board name.